app/assets/v2/js/ptokens/ptokens.js

```diff
@@ -9,6 +9,14 @@ function get_ptokens() {
   return fetchData('/ptokens/', 'GET');
 }
```

Comment:
Does the `get_ptokens()` function work now? If not, or if it's not used anywhere, we should remove that

Reply:
Yes, it was removed.